Jenkinsfile

```diff
@@ -128,12 +147,13 @@ pipeline {
                     steps {
                         sh 'echo $NODE_NAME'
                         sh script: 'echo y | ./script/installation/packages.sh all', label: 'Installing packages'
-
-                        sh script: '''
-                        mkdir build
-                        cd build
-                        cmake -GNinja -DNOISEPAGE_UNITY_BUILD=ON -DNOISEPAGE_TEST_PARALLELISM=1 -DCMAKE_BUILD_TYPE=Debug -DNOISEPAGE_USE_ASAN=ON -DNOISEPAGE_BUILD_BENCHMARKS=OFF -DNOISEPAGE_USE_JUMBOTESTS=OFF ..
-                        ninja''', label: 'Compiling'
+                        
+                        script{
+                            if(!utils){
+                                utils = load("Jenkinsfile-utils.groovy")
+                            }
+                            utils.noisePageBuild(os:"macos",useASAN:true)
+                        }
```

Comment:
nit: Can you run this through a formatter or make sure the spacing matches up with the rest of the file? Maybe something like this:

```
# Build System
script {
    if (!utils) utils = load("Jenkinsfile-utils.groovy")
    utils.noisePageBuild(os:"macos", useASAN:true)
}
```